src/Microsoft.DotNet.Compiler.Common/ProjectContextExtensions.cs

```diff
@@ -37,6 +37,12 @@ public static string GetSDKVersionFile(this ProjectContext context, string confi
             return Path.Combine(intermediatePath, ".SDKVersion");
         }
 
+        public static string IncrementalCacheFile(this ProjectContext context, string configuration, string buildBasePath, string outputPath)
+        {
+            var intermediatePath = context.GetOutputPaths(configuration, buildBasePath, outputPath).IntermediateOutputDirectoryPath;
+            return Path.Combine(intermediatePath, ".IncrementalCache");
```

Comment:
OK - fine with me.  I was just looking to cut down on the number of places these strings are duplicated.